### PR TITLE
add offline installation paragraph & remove important paragraph

### PR DESCRIPTION
**Pull request by WendyYes**

### CODE REVIEW

**Review by chugugrace — APPROVED** [review]

**Comment by markingmyname** [comment]
#sign-off

File: docs/ssdt/release-notes-ssdt-2019.md [hunk 1]
```diff
@@ -21,13 +21,19 @@ monikerRange: ">=aps-pdw-2016||=azuresqldb-current||=azure-sqldw-latest||>=sql-s
 
 Visit https://techcommunity.microsoft.com/t5/SQL-Server-Integration-Services/bg-p/SSIS for the latest information, tips, news, and announcements about SSIS directly from the product team.
 
-## Important
-  - **Version 3.16 is the latest general availability (GA) version, which doesn't support target server version SqlServer2022. Download [SQL Server Integration Services Projects 3.16](https://ssis.gallerycdn.vsassets.io/extensions/ssis/sqlserverintegrationservicesprojects/3.16/1645603822968/Microsoft.DataTools.IntegrationServices.exe).
-More information is available in release notes.**
+## Offline Installation
+Please follow the below steps to install this product in an offline environment:
+1. Follow this [document](https://docs.microsoft.com/en-us/visualstudio/install/create-an-offline-installation-of-visual-studio?view=vs-2019) to create an offline installation of Visual Studio, and make sure following prerequisites are included:
```
Comment (Jak-MS) [inline docs/ssdt/release-notes-ssdt-2019.md:26]:
```suggestion
1. Refer to the instructions in [Create an offline installation package of Visual Studio for local installation](/visualstudio/install/create-an-offline-installation-of-visual-studio?view=vs-2019), and make sure the following prerequisites are included:
```
various MS style and a link edit

File: docs/ssdt/release-notes-ssdt-2019.md [hunk 2]
```diff
@@ -21,13 +21,19 @@ monikerRange: ">=aps-pdw-2016||=azuresqldb-current||=azure-sqldw-latest||>=sql-s
...
+    - Prerequisite Id="Microsoft.VisualStudio.Component.Roslyn.LanguageServices" Version="[16.0,)" DisplayName="C# and Visual Basic"
+    - Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[16.0,)" DisplayName="Visual Studio core editor"
+    - Prerequisite Id="Microsoft.VisualStudio.Component.SQL.SSDT" Version="[16.0,)" DisplayName="SQL Server Data Tools"
+    - Prerequisite Id="Microsoft.Net.Component.4.TargetingPack" Version="[16.0,)" DisplayName=".NET Framework 4 targeting pack"
+    - Prerequisite Id="Microsoft.Net.Component.4.5.TargetingPack" Version="[16.0,)" DisplayName=".NET Framework 4.5 targeting pack"
+    - Prerequisite Id="Microsoft.Net.Component.4.7.TargetingPack" Version="[16.0,)" DisplayName=".NET Framework 4.7 targeting pack"
 
-- **Due to a limitation of VS marketplace, the version 4.1.2 does not introduce new binaries to download. Version 4.1 contains the latest binaries.**
+1. Double click on the installer of this product and perform installation, or you can run the installer in quite mode. Please launch the installer with "/?" argument to get more details of the arguments list of the installer.
```
Comment (Jak-MS) [inline docs/ssdt/release-notes-ssdt-2019.md:34]:
```suggestion
1. Launch the installer of this product and perform the installation, or you can run the installer in quiet mode. Launch the installer with "/?" argument to get more details of the arguments list of the installer.
```

File: docs/ssdt/release-notes-ssdt-2019.md [hunk 3]
```diff
@@ -21,13 +21,19 @@ monikerRange: ">=aps-pdw-2016||=azuresqldb-current||=azure-sqldw-latest||>=sql-s
 
 Visit https://techcommunity.microsoft.com/t5/SQL-Server-Integration-Services/bg-p/SSIS for the latest information, tips, news, and announcements about SSIS directly from the product team.
 
-## Important
-  - **Version 3.16 is the latest general availability (GA) version, which doesn't support target server version SqlServer2022. Download [SQL Server Integration Services Projects 3.16](https://ssis.gallerycdn.vsassets.io/extensions/ssis/sqlserverintegrationservicesprojects/3.16/1645603822968/Microsoft.DataTools.IntegrationServices.exe).
-More information is available in release notes.**
+## Offline installation
+Follow the below steps to install this product in an offline environment:
+1. Refer to the instructions in [Create an offline installation package of Visual Studio for local installation](/visualstudio/install/create-an-offline-installation-of-visual-studio?view=vs-2019), and make sure the following prerequisites are included:
```
Comment (Jak-MS) [inline docs/ssdt/release-notes-ssdt-2019.md:26]:
```suggestion
1. Refer to the instructions in [Create an offline installation package of Visual Studio for local installation](/visualstudio/install/create-an-offline-installation-of-visual-studio?view=vs-2019&preserve-view=true), and make sure the following prerequisites are included:
```

**Pull request merged**